rush.json

```diff
@@ -304,7 +304,7 @@
    * that read these JSON files and do something with them.  These scripts are typically registered
    * in the "eventHooks" section.
    */
-  // "telemetryEnabled": false,
+  "telemetryEnabled": true,
```

Comment:
Just making sure this is intentional

Reply:
We probably shouldn't have this turned on in an OSS repo, even though it doesn't go anywhere.